demo/incubator-avatar-basic-demos.html

```diff
@@ -13,7 +13,7 @@ <h3>Basic usage</h3>
         </incubator-avatar>
         <incubator-avatar name="William Davids">
         </incubator-avatar>
-        <incubator-avatar name="Bruce Williams" abbr="BJ">
+        <incubator-avatar no-tooltip name="Bruce Williams" abbr="BJ" tooltip-position="top" tooltip-align="left">
```

Comment:
What's the point of using tooltip-position and tooltip-align with no-tooltip? Can we make separated demos for positioning?

Reply:
I was planning to change this myself.